.github/workflows/exercises.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         if: matrix.nim != 'devel'
         uses: iffy/install-nim@1ab2fcc1d8294b3956d5696f45cd470c9fe69466
         with:
-          version: "binary:${{ matrix.nim }}"
+          version: "binary:${{ matrix.nim == 'stable' && '1.6.8' || matrix.nim }}"
```

Comment:
I think this should instead be specified as 1.6.8 and referred to as stable. It's not clear that this doesn't get updated when there is a new stable release. That fact is hidden from the matrix definition